pushmanager/templates/pushes.html

```diff
@@ -9,6 +9,16 @@
 <ul id="action-buttons">
 	<li><button id="expand-pushes">Expand All</button></li>
 	<li><button id="collapse-pushes">Collapse All</button></li>
+		&mdash;
+	<li>
+		Show states:
```

Comment:
This Show states appears to be duplicated but the original is certainly not in master. This diff is weird.